lib/jnpr/junos/facts/vmhost.py

```diff
@@ -3,13 +3,42 @@
 from lxml import etree
 
 
+def _get_vmhost_version_information(device):
+    multi_re = False
+    try:
+        rsp = device.rpc.get_route_engine_information(normalize=True)
+        re_list = rsp.findall(".//route-engine")
+        if len(re_list) > 1:
+            multi_re = True
+        else:
+            multi_re = False
+    except RpcError as err:
+        pass
```

Comment:
Please throw the RPC exception, don't pass the exception

Reply:
fixed